code/controllers/subsystems/mapping.dm

```diff
@@ -109,6 +107,10 @@ SUBSYSTEM_DEF(mapping)
 			PRINT_STACK_TRACE("Missing z-level data object for z[num2text(z)]!")
 		level.setup_level_data()
 
+	// Generate turbolifts last!!!
+	for(var/obj/abstract/turbolift_spawner/turbolift as anything in turbolifts_to_initialize)
```

Comment:
Is this something specific for Persistence or could it go upstream eventually?

Reply:
I think it went upstream already?

Reply:
It's not unique to outreach, any dynamically loaded maps like away_sites would have that issue if they had elevators.